backend/lib/pipeline.ml

```diff
@@ -83,11 +86,7 @@ let pipeline ~slack_path ~conninfo ~(info : pr_info) ~dockerfile ~tmpfs
       @ docker_cpuset_mems
     in
     let+ output =
-      Docker.pread ~run_args image
-        ~args:
-          [
-            "/usr/bin/setarch"; "x86_64"; "--addr-no-randomize"; "make"; "bench";
-          ]
+      Docker.pread ~run_args image ~args:(String.split_on_char ' ' cmd)
```

Comment:
Looks like this wouldn't support commands containing quotes. It would be useful to keep `~cmd` as `string list`, at least as input to the pipeline function. Perhaps it would be possible to also get a list from cmdliner, e.g. after a `--`?

Reply:
I agree. I also thought about this and was wondering if there's a better way to do this.

Reply:
We already use a positional argument for the source (e.g. `main.exe local path_to_project`). I think it might be confusing to use `--` and positional options for the command as well.

Reply:
Implemented a custom Cmdliner converter to handle parsing and printing in https://github.com/ocurrent/current-bench/pull/13/commits/27c95b6df3e912433c3005593cfc35360c847a2d. Kept the `--cmd` option.